### PR TITLE
xkill: add Spanish translation

### PR DESCRIPTION
**Pull request by kazulu**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/xkill.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# xkill
+
+> Cierra de manera forzosa una ventana interactivamente en una sesión gráfica.
+> Más información en `kill` y `killall`.
```
Comment (navarroaxel) [inline pages.es/common/xkill.md:4]: En realidad la traducción sería:
```suggestion
> Véase también `kill` y `killall`.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @kazulu, just 1 comment.

Remember that you can send all the translations in 1 PR, we only want 1 commit per file.

**Review by navarroaxel — APPROVED** [review]
LGTM, thanks @kazulu !

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good to me.

**Pull request merged**